### DISCUSSION TITLE
Ajout d'une fonction sur le tableau d'import massif

### DISCUSSION THREAD

**Discussion by Greta712** [betagouv/ma-cantine#1290] [Idées]
Certaines cuisines centrales ne sont pas en mesure de ventiler leurs achats par cantine.
Est-il possible d'intégrer une colonne concernant le nombre de couverts afin de faire la répartition des valeurs en HT en fonction de cette donnée?
@gauthierandre se propose de faire la modification
Vous êtes ok @alemangui et @hfroot ?

**Comment by alemangui** [comment]
Ce serait une simple division arithmétique entre le nombre de couverts et les valeurs ? Si oui, ça peut être fait de leur coté non ?

**Reply by Greta712** [reply]
oui c'est une simple formule mais certains ne sont pas à l'aise avec Excel. A ce jour, je le fais pour ceux qui ont la donnée.

**Reply by gauthierandre** [reply]
Le truc c'est que tout le monde n'a pas forcément besoin de la formule. ça ne concernerait que les cuisines centrales. Ce qu'on peut faire @Greta712 c'est leur configurer le tableau en copiant/collant la formule que je t'avais envoyé quand on fait avec eux leur import ? ou alors on mettre en téléchargement un fichier excel spécifique aux CC?

**Reply by Greta712** [reply]
oui je te rejoins sur ce point c'est spécifique aux CC. La possibilité d'avoir un don déjà prêt à télécharger serait top
